src/Swift.Bindings/tests/IntegrationTests/Swift.Bindings.Integration.Tests.csproj

```diff
@@ -23,7 +23,7 @@
 
     <!-- Compile generated bindings with tests -->
     <ItemGroup>
-      <Compile Include="$(OutputPath)/**/*.cs" Exclude="$(OutputPath)/Metadata/*.cs;$(OutputPath)/ManualBindings/*.cs" />
```

Comment:
Why do we need that change?

Reply:
This cleanup is unrelated to the current PR; I can move it to a separate PR if confusing.

These are old directories used for handling redundant files in the testing infrastructure. The Swift.Runtime files are referenced in the project and emitted by the projection tooling. Conflicts were resolved by removing redundant Swift.Runtime files from the cmake, so this is not needed anymore.

Reply:
ok, thanks for the explanation!